folly/container/F14Map.h

```diff
@@ -1490,4 +1490,28 @@ void swap(
   lhs.swap(rhs);
 }
 
+template <typename K, typename M, typename H, typename E, typename A,
+          typename Pred>
```

Comment:
JFYI internal formatting will translate this to:

```
template <
    typename K,
    typename M,
    typename H,
    typename E,
    typename A,
    typename Pred>
```

Reply:
That's fine. Want me to do it or let clang-format internally do it?

It would be nice to have the `.clang-format` in the open source repo as I asked [last year](https://github.com/facebook/folly/issues/755).

Reply:
I agree that it would be nice! Given that we already do this as of recently with the fbcode_builder scripts, it might be easier to auto-sync `.clang-format` into all the repos now.

Anyway I am just pointing this out up front. No matter what you do, since our `.clang-format` is currently not exported, there would be some little whitespace nit somewhere that internal formatting would change.

Reply:
👍 Thanks for the heads up. No biggie on the whitespace diffs.

I'd love to see the auto syncing with `.clang-format` 😄